plane/source/docs/do-land-start.rst

```diff
@@ -0,0 +1,45 @@
+.. _do-land-start:
+.. _do_land_start:
+
+===================
+Using DO_LAND_START
+===================
+
+The :ref:`DO_LAND_START <mav_cmd_do_land_start>` mission item can be used
+for two purposes:
+
+#. To provide an automatic re-attempt for an aborted NAV_LAND autolanding in a mission. (See :ref:`aborting-autolanding`)
+#. To provide an automatic landing for an RTL.
+
+To trigger an automatic landing as part of an RTL
+(return to launch) you need to do two things:
+
+-  add a :ref:`DO_LAND_START <mav_cmd_do_land_start>`
+   mission item to your mission, just before the start of your landing
+   sequence which setup up the approach waypoints and ends with a NAV_LAND item.
+-  set the :ref:`RTL_AUTOLAND <RTL_AUTOLAND>`
+   parameter to 1 or 2
+
+The way it works is that when the plane enters an RTL it checks to see
+if the parameter RTL_AUTOLAND is set to 1 or 2. If it is then the
+current mission is searched for a mission item of type DO_LAND_START.
+If one is found then the plane will automatically enter AUTO mode and
+land, starting at the part of the mission just after the
+``DO_LAND_START`` marker.
+
+The exact behaviour depends on the :ref:`RTL_AUTOLAND<RTL_AUTOLAND>` value:
+
+-  If :ref:`RTL_AUTOLAND<RTL_AUTOLAND>` = 1, then the plane will first RTL as normal, then
+   when it starts circling the return point (home or a rally point) it
+   will then switch to the AUTO mission after the ``DO_LAND_START`` 
+-  If :ref:`RTL_AUTOLAND<RTL_AUTOLAND>` = 2, then the plane will bypass the RTL completely
+   and go straight to the landing sequence after the ``DO_LAND_START``.
+
+You can optionally include more than one ``DO_LAND_START`` mission item
+in your mission. If that is done then the latitude/longitude of the
+``DO_LAND_START`` mission items is used to choose which landing sequence
+to use. The ``DO_LAND_START`` closest to the current location is used.
```

Comment:
This can be nearest vertically or horizontally I guess. https://github.com/ArduPilot/ardupilot/pull/22147#issuecomment-1314554743